hawc/apps/epiv2/models.py

```diff
@@ -0,0 +1,322 @@
+        StudyPopulation, on_delete=models.CASCADE, related_name="outcomes"
+    )
+    health_outcome = models.CharField(max_length=128,)
+    health_outcome_system = models.CharField(
+        max_length=128,
+        choices=constants.HealthOutcomeSystem.choices,
+        help_text="If multiple cancer types are present, report all types under Cancer.",
+        blank=True,
+        null=True,
+    )
+    created = models.DateTimeField(auto_now_add=True)
+    last_updated = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return self.name
+
+
+class AdjustmentFactor(models.Model):
+    objects = managers.AdjustmentFactorManager()
+
+    name = models.CharField(
+        max_length=64,
+        help_text="A unique name for this adjustment factor that will help you identify it later.",
+    )
+    study_population = models.ForeignKey(
+        StudyPopulation, on_delete=models.CASCADE, related_name="adjustment_factors"
+    )
+    description = models.CharField(max_length=128, blank=True, null=True,)
+    created = models.DateTimeField(auto_now_add=True)
+    last_updated = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return self.name
+
+
+class DataExtraction(models.Model):
+    objects = managers.DataExtractionManager()
+
+    study_population = models.ForeignKey(
+        StudyPopulation, on_delete=models.CASCADE, related_name="data_extractions"
+    )
+    sub_population = models.CharField(max_length=64,)
+    outcome = models.ForeignKey(Outcome, on_delete=models.CASCADE, blank=True, null=True,)
+    exposure_level = models.ForeignKey(
+        ExposureLevel, on_delete=models.SET_NULL, blank=True, null=True
+    )
+    measurement_timing = models.CharField(max_length=128, blank=True, null=True,)
+    n = models.PositiveIntegerField(blank=True, null=True,)
+    effect_estimate_type = models.CharField(
+        max_length=128, choices=constants.EffectEstimateType.choices, blank=True, null=True,
+    )
+    effect_description = models.CharField(
+        max_length=128,
+        blank=True,
+        null=True,
+        verbose_name="Effect estimate description",
+        help_text="Description of the effect estimate with units, including comparison group if applicable",
+    )
+    effect_estimate = models.FloatField(blank=True, null=True,)
+    exposure_rank = models.PositiveSmallIntegerField(
+        blank=True,
+        null=True,
+        help_text="Rank this comparison group by exposure (lowest exposure group = 1)",
+    )
+    ci_lcl = models.FloatField(verbose_name="Confidence Interval LCL", blank=True, null=True,)
+    ci_ucl = models.FloatField(verbose_name="Confidence Interval UCL", blank=True, null=True,)
+    sd_or_se = models.FloatField(
+        verbose_name="Standard Deviation or Standard Error", blank=True, null=True,
+    )
+    pvalue = models.CharField(verbose_name="p-value", max_length=128, blank=True, null=True,)
+    significant = models.BooleanField(
```

Comment:
I see why you did this b/c its the current pattern in hawc but its not working well in other places; maybe better to just use a PositiveSmallIntegerField or CharField w/ choices?